arrow-optics/src/main/kotlin/arrow/optics/extensions/set/at/SetKAt.kt

```diff
@@ -0,0 +1,49 @@
+package arrow.optics.extensions.set.at
+
+import arrow.core.SetK
+import arrow.optics.PLens
+import arrow.optics.extensions.SetKAt
+import kotlin.Any
+import kotlin.Boolean
+import kotlin.Deprecated
+import kotlin.PublishedApi
+import kotlin.Suppress
+import kotlin.jvm.JvmName
+
+@JvmName("at")
+@Suppress(
+  "UNCHECKED_CAST",
+  "USELESS_CAST",
+  "EXTENSION_SHADOWED_BY_MEMBER",
+  "UNUSED_PARAMETER"
+)
+@Deprecated(
+  message = "arrow.optics.extensions package is being deprecated, and it will be removed in 0.13.",
+  level = DeprecationLevel.WARNING
+)
+fun <A, T> PLens<T, T, SetK<A>, SetK<A>>.at(i: A): PLens<T, T, Boolean, Boolean> =
```

Comment:
☝️ 

For all these autogenerated methods I have just tweaked the deprecation message to match what I have seen in #87 . I believe there is no method to replace with, so there is nothing else to do here.